contributed_definitions/nyaml/NXmpes_liquid.yaml

```diff
@@ -365,19 +188,8 @@ NXmpes_liquid(NXmpes):
         solvent(NX_CHAR):
           doc: "Specifies the parent solvent entry of the solution."
     (NXdata):
-      \@signal(NX_CHAR):
-        enumeration: ["data"]
       \@axes(NX_CHAR):
         enumeration: ["energy"]
-      data(NX_NUMBER):
-        \@units(NX_CHAR):
-          exists: recommended
-        \@long_name(NX_CHAR):
```

Comment:
If something is existing in the base class but optional in the base class its more reasonable to leave it out. Was there a specific reason to put this here?

Reply:
Ah yes this was a typo, it was meant like this: `If something is existing in the base class but optional in the application definition its more reasonable to leave it out.`

Reply:
But if it is fine that it aligns with NXmpes it is fine that I removed it.